mapit_gb/data/authorities.json

```diff
@@ -1252,5 +1252,23 @@
   },
   "ards-and-north-down": {
     "gss": "N09000011"
+  },
+  "east-suffolk": {
+    "gss": "E07000244"
+  },
+  "somerset-west-taunton": {
+    "gss": "E07000246"
+  },
+  "west-suffolk": {
+    "gss": "E07000245"
+  },
+  "bournemouth-christchurch-poole": {
+    "gss": "E06000058"
+  },
+  "dorset": {
+    "gss": "E06000059"
+  },
+  "north-lanarkshire": {
+    "gss": "S12000050"
   }
```

Comment:
Should Glasgow also be added?

Reply:
I'm not sure, it seems to the be only one that doesn't have a `govuk_slug` which should be configured when https://github.com/alphagov/mapit/blob/master/mapit_gb/management/commands/mapit_UK_add_slugs_to_local_authorities.py runs, but it seems like it wasn't. So I think this is a different issue that needs fixing.